NineMensMorris/include/gamemanager.h

```diff
@@ -0,0 +1,23 @@
+#ifndef GAMEMANAGER_H
+#define GAMEMANAGER_H
+
+#include <QObject>
+#include <QGraphicsScene>
+#include <QGraphicsView>
+#include "include/menu.h"
+#include "include/game.h"
+
+class GameManager : public QObject
+{
+   Q_OBJECT
+public:
+    QGraphicsScene menuScene, gameScene;
```

Comment:
There is something in the style guidelines about declaring variables on their own line. We can refactor this later